tldw_Server_API/tests/Web_Scraping/test_http_client_fetch.py

```
@@ -131,3 +138,322 @@ def get(self, url, **kwargs):
+        def get(self, url, **kwargs):
+            calls.append((url, kwargs))
+            return DummyResp(
+                url,
+                {"Location": "/final"},
+                status_code=302,
+                text="",
+            )
+
+    monkeypatch.setattr(hc, "_resolve_curl_session", lambda: DummyCurlSession)
+
+    with pytest.raises(ValueError):
```

Ruff (0.15.9)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)